config/examples/zync.yml

```diff
@@ -5,6 +5,7 @@ base: &default
   connect_timeout: 2
   send_timeout: 2
   receive_timeout: 10
+  skip_non_oidc_applications: false
```

Comment:
when you spoke about feature flags, I thought that you meant https://github.com/3scale/porta/blob/master/config/examples/features.yml

I would suggest to use that instead.

Reply:
Yes, I considered this option too, but I didn't like it:
- I found it to be like for bigger kind of "standalone" features. Here it's kind of an option of Zync, so I thought to put it to zync configuration
- it required creating a new class just for loading that config, seemed to be an overkill to me...

Other potential options could be `rolling_updates.yml` or `settings.yml`, but they are kind of a mix of everything, again, I thought it would be better to gather everything zync-related in one file.

Which also means that we need to move `zync_authentication_token` from `settings.yml`, because I think it is the one used at the moment, and not the `authentication.token` in `zync.yml`, we need to clean that up (in another PR).